_playwright-tests/UI/SnapshotRepo.spec.ts

```diff
@@ -102,4 +103,50 @@ test.describe('Snapshot Repositories', () => {
       await expect(row).not.toBeVisible();
     });
   });
+
+  test('test snapshot manual trigger', async ({ page }) => {
```

Comment:
the [test](https://github.com/content-services/content-sources-frontend/pull/531/files#diff-9b23ee601442d5a43ba5875e94378668b283e911eae974f16f2c3babdcd1a1e6R12-R105) above this one just snapshots a repo which was done as part of the PoC and as this is otherwise covered by the repo crud test, I think it would make more sense to just rename it and extend it with the edit and manual trigger [step](https://github.com/content-services/content-sources-frontend/pull/531/files#diff-9b23ee601442d5a43ba5875e94378668b283e911eae974f16f2c3babdcd1a1e6R12-R105) instead of adding a whole new test 😅

Reply:
Done !